bgpd/bgpd.h

```diff
@@ -1861,8 +1861,7 @@ struct bgp_nlri {
 #define BGP_DEFAULT_KEEPALIVE                   60
 #define BGP_DEFAULT_CONNECT_RETRY              120
 
-#define BGP_DEFAULT_EBGP_ROUTEADV                0
-#define BGP_DEFAULT_IBGP_ROUTEADV                0
+#define BGP_DEFAULT_BGP_ROUTEADV 0
```

Comment:
Why this change? Yes, right now the default are the same for iBGP and eBGP, but this may possibly change in the future.

Reply:
Fully agree with @idryzhov. Would be nice adding just an additional option for `advertisement-interval` command to separate iBGP and eBGP timers on-demand.

Reply:
Reverting the changes back,
since as per the above comment,
there is a possibility that we might change the defaults for iBGP and eBGP in future.
Thank you!